src/main/java/com/google/cloud/tools/gradle/appengine/standard/DevAppServerStartTask.java

```diff
@@ -43,13 +45,22 @@ public void setCloudSdkBuilderFactory(CloudSdkBuilderFactory cloudSdkBuilderFact
   @TaskAction
   public void startAction() throws AppEngineException, IOException {
 
+    // Add a listener to write to a file for non-blocking starts, this really only works
+    // when the gradle daemon is running (which is default for newer versions of gradle)
+    File logFile = File.createTempFile("dev_appserver", ".out", getTemporaryDir());
```

Comment:
oh I see what you're saying, that's a good point. It's more for temporary files. I should put this in a real place. brb with this PR